cabal-install/tests/UnitTests/Distribution/Client/Init/NonInteractive.hs

```diff
@@ -323,7 +320,7 @@ driverFunctionTest pkgIx srcDb comp = testGroup "createProject"
               , "[\"test/Main.hs\", \"test/Foo.hs\", \"test/bar.y\"]"
               ]
 
-        case (_runPrompt $ createProject comp silent pkgIx srcDb (emptyFlags {initializeTestSuite = Flag True})) inputs of
+        case (_runPrompt $ createProject comp silent pkgIx srcDb (emptyFlags {initializeTestSuite = Flag True, packageType = Flag LibraryAndExecutable })) inputs of
```

Comment:
This is a woefully long line, it doesn't even fit in github display.

Reply:
If indentation/minimal diff is the problem, I'd got with 1 character indentation to keep the diff small, but break the line.

Reply:
But that's probably not the problem. Perhaps a `let` or `where` with the big argument would help?